service/src/test/java/bio/terra/workspace/service/workspace/ApplicationConfigurationTest.java

```diff
@@ -51,10 +55,19 @@ private void checkLeo(WsmApplication leoApp) {
     assertEquals(leoApp.getState(), WsmApplicationState.OPERATING);
   }
 
-  private void checkCarmen(WsmApplication leoApp) {
-    assertEquals(leoApp.getDisplayName(), "Carmen");
-    assertEquals(leoApp.getDescription(), "musical performance framework");
-    assertEquals(leoApp.getServiceAccount(), "carmen@terra-dev.iam.gserviceaccount.com");
-    assertEquals(leoApp.getState(), WsmApplicationState.DEPRECATED);
+  private void checkCarmen(WsmApplication carmenApp) {
+    assertEquals(carmenApp.getDisplayName(), "Carmen");
```

Comment:
i think assertEquals want expected value in front

Reply:
Good point! switched the rest of these tests too.